streams/src/test/java/org/apache/kafka/streams/processor/TopologyBuilderTest.java

```diff
@@ -17,23 +17,18 @@
 
 package org.apache.kafka.streams.processor;
 
-import static org.junit.Assert.assertEquals;
-
-import static org.apache.kafka.common.utils.Utils.mkSet;
-import static org.apache.kafka.common.utils.Utils.mkList;
-
 import org.apache.kafka.streams.processor.internals.ProcessorNode;
 import org.apache.kafka.streams.processor.internals.ProcessorTopology;
 import org.apache.kafka.test.MockProcessorSupplier;
 import org.apache.kafka.test.MockStateStoreSupplier;
 import org.junit.Test;
 
-import java.util.Collection;
-import java.util.HashMap;
-import java.util.HashSet;
-import java.util.List;
-import java.util.Map;
-import java.util.Set;
+import java.util.*;
```

Comment:
Will change back and update IntelliJ formatting options

> On Nov 21, 2015, at 1:13 PM, Guozhang Wang <notifications@github.com> wrote:
> 
> In streams/src/test/java/org/apache/kafka/streams/processor/TopologyBuilderTest.java:
> 
> >  import org.apache.kafka.streams.processor.internals.ProcessorNode;
> >  import org.apache.kafka.streams.processor.internals.ProcessorTopology;
> >  import org.apache.kafka.test.MockProcessorSupplier;
> >  import org.apache.kafka.test.MockStateStoreSupplier;
> >  import org.junit.Test;
> >  
> > -import java.util.Collection;
> > -import java.util.HashMap;
> > -import java.util.HashSet;
> > -import java.util.List;
> > -import java.util.Map;
> > -import java.util.Set;
> > +import java.util.*;
> We would like to avoid wildcard import in the code base.
> 
> —
> Reply to this email directly or view it on GitHub.
>